CONTRIBUTORS.yaml

```
@@ -2405,6 +2405,10 @@ sitjart:
     affiliations:
       - elixir-europe
 
+skschum:
+    name: Simeon Schum
+    joined: 2024-09
```

not a big deal, but a bit more accurate assuming this will be merged soon

```suggestion
    joined: 2025-02
```